net/netclient/files/netclient.upgrade

```diff
@@ -0,0 +1 @@
+/etc/netclient/
```

Comment:
This is wrong! Entirely wrong. If you want to keep those file during the update. Thats why they are conffiles for.